libs/experimental/langchain_experimental/graph_transformers/llm.py

```diff
@@ -158,6 +159,90 @@ def map_to_base_relationship(rel: Any) -> Relationship:
     )
 
 
+def _convert_to_graph_document(
+    raw_schema: Dict,
+) -> Tuple[List[Node], Tuple[List[Relationship]]]:
+    # If there are validation errors
+    if not raw_schema["parsed"]:
+        try:
+            argument_json = json.loads(
+                raw_schema["raw"].additional_kwargs["tool_calls"][0]["function"][
+                    "arguments"
+                ]
+            )
+            nodes = []
+            for node in argument_json["nodes"]:
+                if not node.get("id"):  # Id is mandatory
+                    continue
+                nodes.append(
+                    Node(
+                        id=node["id"].title(),
+                        type=node.get("type", "Node").capitalize(),
```

Comment:
shouldn't we use the base entity type if there's one set? or do we do that later?